### PR TITLE
added FAQ

### PR DESCRIPTION
**Pull request by sa8**
I added an FAQ file that contains some questions asked to @adlrocha and their answers.
I'm hoping others can contribute too and can use this file when they have a question.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Eudico FAQ
+
```
Comment (adlrocha) [inline FAQ.md:2]: What about adding here a note of the like of: "you can probably find many of these commands in lotus docs [link], but in this FAQ you'll find a list of some that may come handy for the typical user flows needed to interact with Eudico".

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```
+func (t *$YourStruct) MarshalCBOR(w io.Writer) error { return nil }
+
+func (t *$YourStruct) MarshalCBOR(w io.Writer) error { return nil }
+```
+Run  `go run gen.go`. 
+
+
+**Q: How do you send a transaction directly from the code?**
+**A:** By using the `api-MpoolPushMessage.` See it use [here](https://github.com/filecoin-project/eudico/blob/113829e7fc115daac08ea0217170baddcb7788ba/chain/consensus/hierarchical/subnet/manager/manager.go#L375-L391).
```
Comment (adlrocha) [inline FAQ.md:29]:
```suggestion
**A:** By using the `api-MpoolPushMessage.` See how to use it [here](https://github.com/filecoin-project/eudico/blob/113829e7fc115daac08ea0217170baddcb7788ba/chain/consensus/hierarchical/subnet/manager/manager.go#L375-L391).
```

**Review by adlrocha — APPROVED** [review]
@dnkolegov, would you like to include your notes in this PR? If not, @sa8, feel free to merge and we'll keep iterating.

**Comment by dnkolegov** [comment]
I will add later

**Pull request merged**